README.md

```diff
@@ -177,9 +177,11 @@ If the `showClearDates` prop is set to true, an `x` shows up in the input box th
   showClearDates: PropTypes.bool,
 ```
 
-If the `showCalendarIcon` prop is set to true, a calendar icon is displayed at the start of the input box. Interaction is the same as focusing the start date.
+If the `showInputIcon` prop is set to true, a calendar icon is displayed at the start of the input box. Interaction is the same as focusing the start date.
+Optionally, it will display a React node set to `props.customInputIcon`
 ```
-  showCalendarIcon: PropTypes.bool
+  showInputIcon: PropTypes.bool,
+  customInputIcon: PropTypes.node
```

Comment:
> but maybe default to the SVG you've provided.

If it is both optional and customizable it needs to know when to display, when to display the default, and when to display the custom icon.